pkg/issuer/venafi/client/venaficlient_test.go

```diff
@@ -27,6 +27,7 @@ import (
 
 	cmapi "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
 	cmmeta "github.com/jetstack/cert-manager/pkg/apis/meta/v1"
+	"github.com/jetstack/cert-manager/pkg/webhook"
```

Comment:
It feels really ugly that this scheme is still a part of the `webhook` package now that so many other things depend on it... Could this maybe move into `pkg/internal/api` perhaps? (and then be referenced as `internalapi.Scheme`?)

Reply:
I've moved the scheme to pkg/internal/api but I can't import it from there into the cmd/webhook/app/start.go because it's pkg/internal rather than ./internal.
So I've left an alias in place at pkg/webhook